### PR TITLE
Add Engadget icon

### LINKED ISSUE #5309: Engadget
**Name:** Engadget
**Website:** https://www.engadget.com/
**Alexa rank:** 1,241
**Color:** `#2b2d32` or actually a black thing (`#000000`)?
**Official resources for icon and color:** see below on the Engadget's homepage

### PR DESCRIPTION
**Pull request by RPicster**
**Issue:** closes #5309

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I manually extracted the `e` from the engadget svg logo on the website to create the fav icon / small icon described in their logo guide.
I chose `#000` as the default color, `#fff` would have been an alternative, but black fits the general approach of simple icons better.

I also included the margin/whitespace around the `e` as it is requested by the logo guideline:
https://o.aolcdn.com/engadget/brand-kit/eng-logo-guidelines.pdf

### CODE REVIEW

**Comment by RPicster** [comment]
Hm, I see the linter fails because of the size. In the file the viewport size is correct. There is whitespace however - as this is required by the logo guidelines.

Shall I remove the whitespace?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3404,6 +3404,12 @@
             "hex": "72BE50",
             "source": "https://www.empire.kred"
         },
+        {
+            "title": "Engadget",
+            "hex": "000000",
+            "source": "https://www.engadget.com/",
+            "guidelines": "https://o.aolcdn.com/engadget/brand-kit/eng-logo-guidelines.pdf"
```
Comment (LitoMore) [inline _data/simple-icons.json:3411]:
```suggestion
            "source": "https://www.engadget.com",
            "guidelines": "https://o.aolcdn.com/engadget/brand-kit/eng-logo-guidelines.pdf"
```

Let's remove that unnecessary slash.

BTW, could you give more details about the guidelines link? Where can we find this link?

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (RPicster) [reply _data/simple-icons.json]: I removed the trailing slash, although looking at the file, I'd say more than half of the source links have it.
The document was found via the link in this [comment](https://github.com/simple-icons/simple-icons/issues/5309#issuecomment-802714098)

**Comment by LitoMore** [comment]
CleanShot 2022-10-12 at 23 48 58@2x

@RPicster I see some differences. Are you using the resource below as a reference?

```html
<svg class="Fill(engadgetSteelGray) Stk(engadgetSteelGray) Fill(#fff)--md Stk(#fff)--md W(152px) W(120px)--xs Cur(p)" width="33" style="stroke-width:0;vertical-align:bottom" height="33" viewBox="0 0 152 33" data-icon="logo-engadget"><path d="M0 24.1C0 25.7 1.3 27 3 27h12v-3H3v-3h11.8c1.6 0 3.2-1.3 3.2-2.9v.1-5.9c0-1.6-1.5-3.2-3.2-3.2H3c-1.6 0-3 1.6-3 3.2v11.8zM15 18H3v-6h12v6zm40.1-9H43.2c-1.6 0-3.2 1.6-3.2 3.2v11.9-.1c0 1.6 1.6 3 3.2 3H55v3H43v3h12.1c1.6 0 2.9-1.4 2.9-3V12.2c0-1.6-1.3-3.2-2.9-3.2zM55 24H43V12h12v12zM32 9h-8.9c-1.6 0-3.1 1.6-3.1 3.2V27h3V12h9c1.6 0 3 1.5 3 3.2V27h3V15.2c0-3.3-2.7-6.2-6-6.2zm43 15H63v-8.9c0-1.6 1.6-3.2 3.3-3.2H75v15h3V12.2c0-1.6-1.2-3.2-2.8-3.2h-8.9C63 9 60 11.9 60 15.2v8.9c0 1.6 1.7 2.9 3.3 2.9h5.2c2.7 0 5.5-1.1 6.5-3v3m68-27h-3v27h3V12h9V9h-9zm-42 24.1c0 1.6.9 2.9 2.5 2.9H115v3h-11v3h11.4c1.6 0 2.6-1.4 2.6-3.1v.1-17.8c0-1.6-1-3.2-2.6-3.2h-11.9c-1.6 0-2.5 1.6-2.5 3.2v11.9zm14-.1h-11V12h11v12zM95 9H83.4c-1.6 0-3.4 1.6-3.4 3.2v11.9-.1c0 1.6 1.8 3 3.4 3H98V0h-3v9zm0 15H83V12h12v12zm25 .1c0 1.6 1.7 2.9 3.3 2.9H135v-3h-12v-3h12.2c1.6 0 2.8-1.3 2.8-2.9v.1-5.9c0-1.6-1.2-3.2-2.8-3.2h-11.9c-1.6 0-3.3 1.6-3.3 3.2v11.8zm15-6.1h-12v-6h12v6z"></path></svg>
```

**Comment by RPicster** [comment]
Hm, yes there is a significant difference.
I repeated the whole process with the source you linked (which is the svg from the sites top).
I also rebased to develop.

**Comment by LitoMore** [comment]
CleanShot 2022-10-15 at 03 19 00@2x

@RPicster I still see some differences. But I think it's acceptable due to the source resource is smaller than our viewbox.

**Review by LitoMore — APPROVED** [review]

**Pull request merged**